vite.config.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  build: {
+    assetsInlineLimit: 100000000,
+    rollupOptions: {
+      output: {
+        assetFileNames: 'index.[ext]',
+        entryFileNames: 'index.js',
+      }
+    }
+  },
+  base: "/DmitriyIshchenko/eCommerce-App/",
```

Comment:
I don't think we need to specify base path like this anymore, since we aren't working with school repo.

This should be fine:
```suggestion
  base: "/",
```